controller/array_action/array_mediator_interface.py

```diff
@@ -240,8 +242,7 @@ def create_snapshot(self, volume_id, snapshot_name, space_efficiency, pool):
             Snapshot
         Raises:
             ObjectNotFound
-            IllegalObjectName
-            IllegalObjectID
+            InvalidArgument
```

Comment:
when the user specifies space efficiency in snapshot class, in order to remain backward compatible, I think we should use old fcmaps

Reply:
and incompatible pool

Reply:
and on create volume with space efficiency that is different from its source